pkgs/by-name/in/invidious-router/package.nix

```
@@ -0,0 +1,27 @@
+{ lib, buildGo122Module, fetchFromGitLab }:
+let
+  pname = "invidious-router";
+  version = "1.0";
+in
+buildGo122Module {
+  inherit pname version;
+
+  src = fetchFromGitLab {
+    owner = "gaincoder";
+    repo = "invidious-router";
+    rev = version;
+    hash = "sha256-6apw+UnhDSuPa9QB3S8b8Ej3NJTK+UCdbDjC2LkhLIg=";
+  };
+
+  vendorHash = "sha256-c03vYidm8SkoesRVQZdg/bCp9LIpdTmpXdfwInlHBKk=";
+
+  doCheck = true;
+
+  meta = {
+    homepage = "https://gitlab.com/gaincoder/invidious-router/";
+    description = "A Go application that routes requests to different Invidious instances based on their health status and (optional) response time";
```

I think something more concise like
```suggestion
    description = "Route requests to different Invidious instances based on their health status and response time";
```
would be better but I leave it up to you.

---

This is the official description, I'd like to stick to it